src/ui/index.tsx

```diff
@@ -102,7 +104,10 @@ new BackgroundScriptUpdateHandler({
   onFailedHandshake: () => handleFailedHandshake(),
 }).keepAlive();
 
-initializeUI();
+initializeUI().then(() => {
+  // TODO: remove this event?
+  emitter.emit('uiInitialized');
```

Comment:
Looks like it's not being used yet... but it's just an event, it's not hurting anyone :)

Reply:
thanks!